src/api/participants/handler.js

```diff
@@ -102,9 +102,18 @@ const create = async function (request, h) {
         }
       }
     } else {
-      const participantCurrencyId1 = await ParticipantService.createParticipantCurrency(participant.participantId, request.payload.currency, ledgerAccountTypes.POSITION, false)
-      const participantCurrencyId2 = await ParticipantService.createParticipantCurrency(participant.participantId, request.payload.currency, ledgerAccountTypes.SETTLEMENT, false)
-      participant.currencyList = [await ParticipantService.getParticipantCurrencyById(participantCurrencyId1), await ParticipantService.getParticipantCurrencyById(participantCurrencyId2)]
+      const defaultSettlementModels = allSettlementModels.filter(model => model.currencyId === null)
```

Comment:
This case, where there is only 1 settlement model looks very similar to the above case, where there is more than 1 settlement model. Is there a way we can make it more generic and easier to read?